src/map/skill.cpp

```diff
@@ -23059,6 +23092,12 @@ void do_init_skill(void)
 
 void do_final_skill(void)
 {
+	skill_db.clear();
+	abra_db.clear();
+	magic_mushroom_db.clear();
+	reading_spellbook_db.clear();
+	skill_arrow_db.clear();
```

Comment:
Do we ever call `do_final_skill` outside of closing the map server? If not, we don't have to do these additions as it will be handled automagically.

Reply:
I guess it doesnt hurt and we added it for all others dbs, wherever we did not forget about it.